app/controllers/admin/recommendations_controller.rb

```diff
@@ -28,23 +35,41 @@ def add_recommendation
   end
 
   def edit_recommendation
-    recommendation = Recommendation.find(params[:recommendation_id])
-
-    book_ids = params[:book_ids]
-    school = params[:school]
-    organization = params[:organization]
-    country = params[:country]
-
-    recommendation.books = []
-    book_ids.each do |book_id|
-      recommendation.books << Book.find(book_id)
+    recommendation = Recommendation.find(params[:id])
+    recommendation_type = params[:recommendation_type].to_i
+    recommendation.recommendation_type = recommendation_type
+    recommendation.name = params[:name]
+
+    if recommendation_type == 0
```

Comment:
this seems to be pretty much the same as the `create_recommendations` so maybe make a helper function?